csrc/dynamic_transform.cpp

```diff
@@ -660,6 +665,7 @@ class DynamicTransformConcretizer : public OptOutMutator {
   //! Use this instead of calling registerMutation directly, since it will also
   //! check that the concretized value is a valid input to all of its uses.
   void registerConcretization(Val* old_val, Val* new_val) {
+    symbolic_to_concretized_map_.emplace(old_val, new_val);
```

Comment:
True. I decide to place all mutations in a single map.

Reply:
I did consider returning `DynamicTransformConcretizer` itself. Then, you wouldn't have to add this ^^^ mutation to `symbolic_to_concretized_map_`.

Reply:
I just meant that you could just return `mutations_` instead of `symbolic_to_concretized_map_` in `getSymbolicToConcretizedMap()` since `registerMutation` essentially just does `mutations_[old_val] = new_val` anyway. https://github.com/NVIDIA/Fuser/blob/0cc22b910b5a6302444850019ae1e4f92974bc83/csrc/mutator.cpp#L80